### PR TITLE
refactor PHP 8.1

### PR DESCRIPTION
**Pull request by yekovalenkoa**

### CODE REVIEW

**Review by Wiktor6 — APPROVED** [review]

File: composer.json [hunk 1]
```diff
@@ -1,50 +1,53 @@
...
+        "ext-json": "*",
+        "guzzlehttp/guzzle": "^6.0",
+        "phpstan/phpstan-deprecation-rules": "^1.1",
+        "symfony/http-kernel": "^6.0",
+        "symfony/serializer": "^6.0",
+        "webmozart/assert": "^1.11"
+    },
+    "require-dev": {
+        "roave/security-advisories": "dev-master",
+        "phpunit/phpunit": "^9.6.16",
```
Comment (lukasz-falda) [inline composer.json]: maybe you can bump to 10?

File: phpstan.neon [hunk 2]
```diff
@@ -1,7 +1,14 @@
...
+    checkNullables: true
+    checkUnionTypes: true
+    treatPhpDocTypesAsCertain: false
     paths:
-        - %rootDir%/../../../src
\ No newline at end of file
+        - %rootDir%/../../../src
+    excludePaths:
+        - src/Tests/*
+        - tests/*
```
Comment (lukasz-falda) [inline phpstan.neon:13]: don't exclude tests paths if you don't have to
#whatever

File: tests/Integration/Command/FindPointsTest.php [hunk 3]
```diff
@@ -94,7 +94,7 @@ private function getCommand(): FindPoints
         return new FindPoints($this->client, new Serializer());
     }
 
-    private function getSuccessfulBody(): string
+    private function getSuccessfulBody(): string|false
```
Comment (lukasz-falda) [inline tests/Integration/Command/FindPointsTest.php]: If we expect string as return leave string only.

You can throw exception on method if you wish

**Review by lukasz-falda — COMMENTED** [review]
this will be a new major version, right?

File: composer.json (hunk 1 above)
Comment (yekovalenkoa) [reply composer.json]: Done :)

File: tests/Integration/Command/FindPointsTest.php (hunk 3 above)
Comment (yekovalenkoa) [reply tests/Integration/Command/FindPointsTest.php]: Done :)

**Review by lukasz-falda — APPROVED** [review]

**Pull request merged**